{{ cookiecutter.project_slug }}/.buildkite/pipeline.yml

```diff
@@ -0,0 +1,10 @@
+env:
+  CI: true
+  TF_IN_AUTOMATION: true
+
+steps:
+  - label: ":terraform: Terraform Format"
+    agents:
+      queue: "{{ cookiecutter.buildkite_agent }}"
+    commands:
+      - make fmt
```

Comment:
If this is meant to be a test of sorts I wouldn't have it do the actual fmt I'd have it run fmt -check -recursive and then evaluate the response and fail if it's not formatted.

Reply:
I think you may be looking at an individual commit, not the whole PR as that is exactly what it does.  It does a 'terraform fmt -recursive -check -diff -write=false' which will print out the unformatted lines and return a non-zero response code if its not formatted.

Reply:
Yeah. Sorry my bad.